tests/components/modbus/conftest.py

```diff
@@ -80,6 +80,7 @@ async def base_test(
     config_modbus=None,
     scan_interval=None,
     expect_init_to_fail=False,
```

Comment:
We need `expect_init_to_fail` for only one test for the modbus cover:
```
    if discovery_info is None:
        _LOGGER.warning(
            "You're trying to init Modbus Cover in an unsupported way."
            " Check https://www.home-assistant.io/integrations/modbus/#configuring-platform-cover"
            " and fix your configuration"
        )
        return
```
I wonder if we can remove the warning and the tests and keep the conftest simpler